spec/models/concerns/planning_application/validation_request_spec.rb

```diff
@@ -34,24 +34,52 @@
     let(:planning_application) { create(:planning_application, :invalidated) }
 
     let!(:red_line_boundary_change_validation_request) do
-      create(:red_line_boundary_change_validation_request, planning_application: planning_application)
+      create(
+        :red_line_boundary_change_validation_request,
+        planning_application: planning_application,
+        created_at: 1.day.ago
+      )
     end
+
     let!(:post_validation_red_line_boundary_change_validation_request) do
       create(:red_line_boundary_change_validation_request, :post_validation, planning_application: planning_application)
     end
+
     let!(:other_change_validation_request) do
-      create(:other_change_validation_request, planning_application: planning_application)
+      create(
+        :other_change_validation_request,
+        planning_application: planning_application,
+        created_at: 2.days.ago
+      )
     end
+
     let!(:replacement_document_validation_request) do
-      create(:replacement_document_validation_request, planning_application: planning_application)
+      create(
+        :replacement_document_validation_request,
+        planning_application: planning_application,
+        created_at: 3.days.ago
+      )
     end
+
     let!(:additional_document_validation_request) do
-      create(:additional_document_validation_request, planning_application: planning_application)
+      create(
+        :additional_document_validation_request,
+        planning_application: planning_application,
+        created_at: 4.days.ago
+      )
+    end
+
+    let!(:description_change_validation_request) do
+      create(
+        :description_change_validation_request,
+        planning_application: planning_application,
+        created_at: 5.days.ago
+      )
     end
 
     context "when no argument is supplied" do
       it "returns validation requests where post_validation is false" do
-        expect(planning_application.validation_requests).to match_array(
+        expect(planning_application.validation_requests).to eq(
```

Comment:
I noticed we're ordering by `created_at`, so seemed worth checking the order as well as the contents of the array.